src/components/CreateAreaModal/CreateAreaModal.jsx

```diff
@@ -70,6 +73,15 @@ const CreateAreaModal = ({ isOpen, setIsOpen }) => {
                 required
               />
             </Form.Group>
+            <Form.Group className="mb-3" controlId="createArea.state">
```

Comment:
If I don't press any state and then create a new area, it passes an empty string to the backend instead of Alabama. Maybe add a "select state" option to force the user to click the dropdown?

![image](https://user-images.githubusercontent.com/57849152/166341213-d18145b2-f29f-4fe3-8b5b-6068117566f8.png)

Reply:
Ah so state is a required field?

Reply:
yes